.ng-dev/release.ts

```diff
@@ -1,20 +1,66 @@
-import {join} from 'path';
-import {ReleaseConfig} from '@angular/dev-infra-private/release/config';
-import {releasePackages} from '../tools/release/release-output/release-packages';
-import {promptAndGenerateChangelog} from '../tools/release/changelog';
+import {BuiltPackage, ReleaseConfig} from '@angular/dev-infra-private/release/config';
+import {ReleaseAction} from '@angular/dev-infra-private/release/publish/actions';
+import {SemVer} from 'semver';
+
+import {
+  assertValidFrameworkPeerDependency
+} from '../tools/release-checks/check-framework-peer-dependency';
+import {
+  assertValidUpdateMigrationCollections
+} from '../tools/release-checks/check-migration-collections';
+import {assertValidNpmPackageOutput} from '../tools/release-checks/npm-package-output';
+
+const actionProto = ReleaseAction.prototype as any;
+const _origStageFn = actionProto.stageVersionForBranchAndCreatePullRequest;
+const _origVerifyFn = actionProto._verifyPackageVersions;
+
+// Patches the `@angular/dev-infra-private` release tool to perform sanity checks
+// before staging a release. This is temporary until the dev-infra team has implemented
+// a more generic solution to running sanity checks before releasing (potentially building
+// some of the checks we have in the components repository into the release tool).
+actionProto.stageVersionForBranchAndCreatePullRequest = async function(newVersion: SemVer) {
+  await assertValidFrameworkPeerDependency(newVersion);
+  await assertValidUpdateMigrationCollections(newVersion);
+
+  return await _origStageFn.apply(this, arguments);
+};
+
+// Patches the `@angular/dev-infra-private` release tool to perform sanity
+// checks of the NPM package output, before publishing to NPM.
+actionProto._verifyPackageVersions =
+    async function(newVersion: SemVer, builtPackages: BuiltPackage[]) {
+  await assertValidNpmPackageOutput(builtPackages, newVersion);
+
+  return await _origVerifyFn.apply(this, arguments);
+};
+
+/**
+ * Packages that will be published as part of the project.
+ *
+ * Note: The order of packages here will control how sections
+ * appear in the changelog.
+ */
+export const releasePackages = [
```

Comment:
It seems like this is repeated with `tools\release\release-output\release-packages.ts`.

Reply:
This file has been removed so that the config in `.ng-dev/release.ts` is the primary list of packages.